src/networking.c

```diff
@@ -2029,18 +2030,32 @@ void sendReplyToClient(connection *conn) {
     writeToClient(c, 1);
 }
 
+static inline int canFsyncUsingIOUring(void) {
```

Comment:
I feel that it is important to call out the `always` part.

```suggestion
static inline int canUseIoUringForAlwaysFsync(void) {
```